packages/webview_flutter/example/integration_test/webview_flutter_test.dart

```diff
@@ -1272,18 +1276,19 @@ void main() {
       ),
     );
     final WebViewController controller = await controllerCompleter.future;
-    await controller.evaluateJavascript('window.open("about:blank", "_blank")');
+    await controller
+        .evaluateJavascript('window.open("https://flutter.dev/", "_blank")');
```

Comment:
This test would stall on Android. For some reason, Android has different behavior when a `window.open` is called with a url with `https://` vs not. It loads normally when using a URL with `https://`, but appends the url to the current url when it doesnt. (e.g. `https://flutter.dev/` turns into `https://flutter.dev/about:blank`)